tests/src/core/testqgslayoutitemmap.cpp

```diff
@@ -0,0 +1,85 @@
+/***************************************************************************
+                         testqgslayoutitemmap.cpp
```

Comment:
There's already testqgslayoutmap.cpp, please add to that file instead of making a new one.

Reply:
Is it possible to run a single test class via cmake, if do that ?

Reply:
Also, that testqgslayout.cpp has ~1300 lines already, and a single class, why do we don't want a per-class testcase ?

Reply:
Add it to testqgslayoutmap , not testqgslayout. Or the python equivalent.

Reply:
Yes, now I realyze that test is specific to QgsLayoutItemMap (should it be renamed to match with class name?)